app/modules/file.c

```diff
@@ -13,6 +13,14 @@
 
 #define FILE_READ_CHUNK 1024
 
+// use this time/date in absence of a timestamp
+#define FILE_TIMEDEF_YEAR 2016
```

Comment:
Can I recommend using a more common value for the epoch? E.g. the Unix epoch (Jan 1, 1970) or ZigBee (Jan 1, 2000).